### PR TITLE
change some Devise params

### PR DESCRIPTION
**Pull request by sseerrggii**
As explained on #454 we will try to change this Devise parameters.

I will need to test it on staging and staging app

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -130,7 +130,7 @@
   config.remember_for = 4.weeks
 
   # If true, extends the user's remember period when remembered via cookie.
```
Comment (sauloperez) [inline config/initializers/devise.rb]: Do we remember via cookie?

Comment (sseerrggii) [reply config/initializers/devise.rb]: Yes

**Comment by markets** [comment]
@sseerrggii @sauloperez I pushed this change in order to fix the CI: https://github.com/coopdevs/timeoverflow/pull/456/commits/dd0e4875e45f2fadda22d97c8078b5179c205711. 

We should re-test this thing :pray: #432

**Comment by enricostano** [comment]
Deployed to staging. cc/ @sseerrggii

**Comment by sseerrggii** [comment]
Sadly don't work as expected 

`config.sign_out_all_scopes` is true but when I sign out session started without 'remember me' the `REMEMBER CREATED AT` is reset

Also I don't see that `config.extend_remember_period` refresh date on `REMEMBER CREATED AT`

Actually I'm not sure what is the behaviour expected of changing this variables. I didn't find it in Devise documentation https://www.rubydoc.info/github/plataformatec/devise/Devise/Models/Rememberable

**Comment by enricostano** [comment]
@sseerrggii what happened? Not needed any more?

**Comment by sseerrggii** [comment]
Sorry I didn't explain why :stuck_out_tongue_winking_eye: 
This PR is old and I want to do a new PR with a diferent approach to the problem

**Pull request closed without merging**